Automattic/wp-calypso
# A4A: reuse language package

Resolves https://github.com/Automattic/automattic-for-agencies-dev/issues/692

## Proposed Changes

Re-use existing language package for the Partner Directory form

<img width="702" alt="Screenshot 2024-06-18 at 7 52 13 AM" src="https://github.com/Automattic/wp-calypso/assets/60262784/bf7720c8-01ab-4312-8f48-beaaa40c4ade">


## Why are these changes being made?



## Testing Instructions



- Navigate to `/partner-directory/agency-details` and check the language selector.

## Pre-merge Checklist
